content/en/docs/ambient/usage/ztunnel/index.md

```diff
@@ -475,7 +457,7 @@ In the use cases so far, the traffic source and destination pods are both ambien
 
 ### East-West non-mesh pod to ambient mesh pod (and use of PeerAuthentication resource) {#ewnonmesh}
 
-In the example below, the same `httpbin` service which has already been set up in the prior examples is accessed via client `sleep` pods that are running in a separate namespace that is not part of the Istio mesh. This example shows that East-west traffic between ambient mesh pods and non mesh pods is seamlessly supported. Note that as described previously, this use case leverages the traffic hair-pinning capability of ambient. Since the non-mesh pods initiate traffic directly to the backend pods without going through HBONE or ztunnel, at the destination node, traffic is redirected via the ztunnel proxy at the destination node to ensure that ambient authorization policy is applied (this can be verified by viewing logs of the appropriate ztunnel proxy pod on the destination node; the logs are not shown in the example snippet below for simplicity).
+In the example below, the same `httpbin` service which has already been set up in the prior examples is accessed via client `sleep` pods that are running in a separate namespace that is not part of the Istio mesh. This example shows that East-west traffic between ambient mesh pods and non mesh pods is seamlessly supported. Since the non-mesh pods initiate traffic directly to the backend pods without going through HBONE or ztunnel. Note that in this case, Istio L4 policy can control whether traffic is denied or allowed to bypass the waypoint.
```

Comment:
Yeah, seems clearer to me. I'd remove the preceding sentence at this point too though.
```suggestion
In the example below, the same `httpbin` service which has already been set up in the prior examples is accessed via client `sleep` pods that are running in a separate namespace that is not part of the Istio mesh. This example shows that East-west traffic between ambient mesh pods and non mesh pods is seamlessly supported. Note that in this case, Istio L4 policy can control whether traffic is denied or allowed to bypass the waypoint.
```

Reply:
Hi @MorrisLaw @ilrudie I think I'm confused with `Istio L4 policy can control whether traffic is denied or allowed to bypass the waypoint.`  My mental model is this is controlled by source ztunnel based on the source (out of mesh or ingress or sidecar or ztunnel captured pod).  You have L4 policy which is enforced at destination ztunnel thus it is confusing to me that it controls bypass waypoint. I think it should be:

Note that in this case, Istio L4 policy can control whether traffic is denied or allowed from the source (which is sleep here I think?).

Reply:
Is this policy part, specifically, documented somewhere @linsun ?

>  Istio L4 policy can control whether traffic is denied or allowed from the source

I'm getting a bit confused myself now. I think a reference might be helpful.

Reply:
That makes sense, thank you for clarifying!

Reply:
Updated, PTAL @linsun

Reply:
Source ztunnel does control if traffic is sent to a waypoint or not but it's not via an authz policy. The destination ztunnel can enforce things like "this wl can only be reached from my waypoint's identity" or conversely it can be permissive of traffic without an identity or it may allow an identity of a non-waypoint wl to reach the destination.

I was describing that as allowing the waypoint to be "bypassed". I can see how that's a confusing way to think about/describe what's happening though.